activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -84,6 +84,15 @@ def sql_color(sql)
       def logger
         ActiveRecord::Base.logger
       end
+
+      def debug(progname = nil, &block)
+        backtrace = Rails.backtrace_cleaner.clean caller
```

Comment:
@rafaelfranca You mean re-implement an `_extract_callstack` private method the same way the Reporting class does?